omero/conf.py

```diff
@@ -46,10 +46,9 @@
     release = 'UNKNOWN'
 
 rst_prolog = """
-.. note:: **This documentation is for the new OMERO 5.4 version.** See the `latest OMERO 5.3.x
-    version <http://www.openmicroscopy.org/site/support/omero5.3/>`_ or the
-    :legacy_plone:`previous versions <>` page to find documentation for the
-    OMERO version you are using if you have not upgraded yet.
+.. note:: **This documentation is for the new OMERO 5.4 version.** See the
```

Comment:
I updated this for now but then I see @sbesson has suggested removing it and adding the docs index link to the side bar instead - certainly open to doing that